mob.go

```diff
@@ -998,6 +999,11 @@ func done(configuration Configuration) {
 		}
 
 		git("branch", "-D", wipBranch.Name)
+
+		if uncommittedChanges && !configuration.MobDoneSquash { // give the user the chance to name their final commit
+			silentgit("reset", "--soft", "HEAD^")
```

Comment:
I think this should not be a silentgit but a git as this is helpful for the user to see.